.travis/install.sh

```diff
@@ -46,6 +46,10 @@ elif [ -n "${LIBRESSL}" ]; then
     fi
 fi
 
+if [ -n "${DOCKER}" ]; then
+    docker pull "$DOCKER"
```

Comment:
Can we add an assert that there's no `$OPENSSL` or `$LIBRESSL` when `$DOCKER` is set?

Reply:
So something like `if [ -n "${OPENSSL}" ] || [ -n "${LIBRESSL}" ]; then echo "Can't set those things"; exit 1; fi`

Reply:
sure, that's what passes for an assert in this godforesaken language